extensions/sql-migration/src/telemtery.ts

```diff
@@ -41,6 +46,10 @@ export enum TelemetryAction {
 	Cancel = 'cancel',
 }
 
+export function logError(telemetryView: TelemetryViews, err: any): void {
```

Comment:
err should be a string here - this doesn't support passing in error objects (which is what you're still doing in some places)